src/controllers/cart.js

```diff
@@ -23,14 +23,14 @@ exports.postCart = async (req, res, next) => {
   }
 };
 
-exports.deleteCart = async (req, res) => {
+exports.deleteCart = async (req, res, next) => {
   try {
     Cart.deleteOne({ customerId: req.user.id });
-    res.status(204).json({
+    return res.status(204).json({
```

Comment:
You don't have to add message in response since status code 204 nothing will be displayed in response
instead of that use : return res.sendStatus(204);